src/services/rocketChat/services/rocketChatLogin.js

```diff
@@ -31,6 +36,7 @@ class RocketChatLogin {
 					const res = await request(
 						getRequestOptions('/api/v1/me', {}, false, { authToken, userId: rcAccount.rcId }, 'GET')
 					);
+ 					await setUserStatus({authToken, userId: rcAccount.rcId}, 'offline');
 					if (res.success) return { authToken };
 				} catch (err) {
```

Comment:
Currently the error is catched but nothing is logged. We should at least add logging there.